examples/mnist-mlp/Main.hs

```diff
@@ -91,6 +93,8 @@ main = do
   model <- foldLoop init 5 $ \model _ ->
     runContT (streamFromMap (datasetOpts 2) trainMnist) $ trainLoop localDevice model optimizer . fst
 
+  performGC
+  dumpLibtorchObjects 1
```

Comment:
I would remove the code from the example before merging since this probably confuses new users who are just looking to see an mnist example for the first time.

Reply:
if you want to keep this for testing/diagnostics, would copy the example to the experimental directory as its own program.

Reply:
Thank you for your reviewing.
You are right. It should be removed on the example.